target_actionkit/sinks.py

```diff
@@ -7,7 +7,7 @@ class ContactsSink(ActionKitSink):
     """ActionKit target sink class."""
 
     name = "Contacts"
-    endpoint = "user"
+    endpoint = "rest/v1/"
```

Comment:
Was this intentional?

Reply:
Yes! Without it I would not be able to add users to the action page